charts/jenkins/values.yaml

```diff
@@ -388,6 +388,10 @@ controller:
 
   terminationGracePeriodSeconds:
 
+  terminationMessage:
```

Comment:
Flatten this object please, no need for it

also I don’t think we should be setting a value, just leave it empty and we only set it if there’s a value in it

Reply:
Thanks for the feedback! I've flattened the properties and left the values empty by default. I've adjusted the documentation and updated the unit tests.